services/hexathons/src/models/visit.ts

```diff
@@ -0,0 +1,54 @@
+import { accessibleRecordsPlugin, AccessibleRecordModel } from "@casl/mongoose";
+import mongoose, { Schema, Types, model } from "mongoose";
+
+export interface Visit extends mongoose.Document {
+  visitorId: string;
+  hexathon: Types.ObjectId;
+  company: Types.ObjectId;
+  employees: string[];
```

Comment:
why would employees be an array here?

Reply:
That was how it was originally. I interpreted it as multiple employees could've met with an attendee during a "visit"

Reply:
ah ok